modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -230,11 +221,17 @@ include::example$StartUsing.java[tags=connect-env,indent=0]
 --
 ====
 
-Following successful authentication, add this code snippet to access your `Bucket`:
+TIP: There's also a simpler version of `Cluster.connect()` for when you don't need to customize the cluster environment:
```

Comment:
I think you want the snippet in the TIP?
In Antora you can do that with:

[TIP]
.Simpler Connection
====
There's also a simpler version of `Cluster.connect()` for when you don't need to customize the cluster environment:

[source,java]
----
include::devguide:example$SimpleConnect.java[tags=connect-string,indent=0]
----
====

Reply:
Thanks! Done.